### DISCUSSION TITLE
Photos go missing

### DISCUSSION THREAD

**Discussion by Freetodream** [photoprism/photoprism#1891] [General]
Hello everyone.
This is my first time using photoprism.
I have a raspberry pi4 and successfully set up the photoprism server, with original and storage folders set to an external HHD, auto start on reboot.
Everything was going very well until  I removed the external HHD and forgot to put it back after turning on the raspberry pi again.
When I accessed the photoprism webapp all my photos appeared with he broken thumbnail then disappeared. I tried indexing/rescan but the photos do not show up. Photos are still found in the original folder.
New photos can be uploaded with no issue.

Please help.

**Comment by lastzero** [comment]
You reconnected the external hard drive and mounted the Folders as originally. You rescanned the library. But everything remains gone? Did you try restarting the computer and containers?

**Reply by Freetodream** [reply]
Yes the external HHD was reconnected. The folder are mounted (also new photos will be uploaded to original folder as expected).
I restarted the raspberry pi with no result.
I rescanned the library with no result.
I assume restarting container with sudo docker-compose up -d. No result. 
Any other thoughts on the problem?

**Reply by lastzero** [reply]
It should find the files again. A lot of guesswork for us at this point and it's already past 9pm...

**Reply by Freetodream** [reply]
Ok thanks for your suggestions

**Reply by Freetodream** [reply]
Maybe this might be useful:

The way I installed photoprism on my raspberrypi
mkdir Photoprism 
cd Photoprism
wget https://dl.photoprism.org/docker/arm64/docker-compose.yml
sudo docker pull --platform=arm64 photoprism/photoprism:latest
nano docker-compose.yml
//changed password.
sudo docker-compose up -d

Then i set up auto start:
sudo systemctl enable docker
cd Photoprism
nano docker-compose.yml
sudo docker-compose up -d


All good so far
I upload files and they end up in the picture folder.
Lets call this (Instance_number1)

If I restart my raspberrypi, then the photos disappear. When I upload, the photos are not uploaded to the picture folder, but i presume somewhere else (which i can't find). Lets call this (Instance_number2)

If I execute: 
cd Photoprism
sudo docker-compose stop
sudo docker-compose up -d
Then my photos are back again and uplaods go to the picture folder once again. 
i.e. back to (Instance_number1)

If i restart again then photoprism auto starts for me but im back to (Instance_number2). 
- Again can't figure out where iti s uploading my files but they are still there somewhere. 

If I execute again the following: 
cd Photoprism
sudo docker-compose stop
sudo docker-compose up -d
Then my photos are back again and uplaods go to the picture folder once again. 
i.e. back to (Instance_number1)

**Reply by Freetodream** [reply]
Solution:
My mount point for harddrive (before drive is mounted) was being locally written to. (Had to imitate this mount point to fix the problem) When the HDD was mounted then this was written to if photoprism restarted. This made it look like two instances.
